downstream/modules/automation-hub/proc-push-container.adoc

```diff
@@ -25,9 +25,10 @@ $ podman login -u=[username] -p=[password] [automation-hub-url]
 . Push your container image to your {HubName} container registry:
 +
 -----
-$ podman push [automation-hub-url]/[container image name]
+$ podman push [automation-hub-url]/[container image name] --remove-signatures
 -----
-
++
+NOTE: The `--remove-signatures` flag is required when signed images from registry.redhat.io are pushed to the {HubName} container registry. The `push` operation re-compresses image layers during the upload, which is not guaranteed to be reproducible and is client implementation dependent. The `--remove-signatures` flag prevents possible image-layer digest changes during the `push` operation that will cause the operation to fail.
```

Comment:
The last sentence conceptually is not correct, unfortunately. The flag does not prevent image layers digest change. If it would be up to me, I'd include the error message as I was initially suggesting. It might happen, the customer would not familiarize themselves with docs ahead of push operation and get this error. Seeing the error in docs can save from customer cases :)